cmd/support/main.go

```diff
@@ -280,19 +305,56 @@ func getK8sResource(resource string, dest string) {
 			logrus.Warnf("Failed to get %s", resource)
 		}
 		// We still want to write the output if the resource was not found or another issue occured as that can shed info on whats going on
-		_ = os.WriteFile(fmt.Sprintf("%s/%s.log", dest, resource), out, os.ModePerm)
+		_ = os.WriteFile(fmt.Sprintf("%s/%s-resource.log", dest, resource), out, os.ModePerm)
 	}
 }
 
-// copyFilesInDir copies all files in the source dir to destination path, keeping the name. Follows dirs inside dirs.
-func copyFilesInDir(sourceDir string, destpath string) {
-	destDir := path.Base(destpath)
-	fullDestDir := fmt.Sprintf("%s/%s", destpath, destDir)
-	_ = os.MkdirAll(fullDestDir, os.ModeDir)
-	cmd := exec.Command("cp", "--recursive", sourceDir, fullDestDir)
-	_, err := cmd.CombinedOutput()
+// getK8sPodsLogs gets the logs of the given resource on the given namespace, unless the resource is empty, in which case it
+// will get ALL the logs for ALL the pods in a given namespace
+func getK8sPodsLogs(resource, namespace, dest string) {
+	var podNames []string
+	kubectlconfig, err := getKubeConfig()
 	if err != nil {
-		logrus.Warnf("Failed to copy %s to %s", sourceDir, fullDestDir)
+		logrus.Warnf("Could not get kubeconfig, skipping k8s info gathering: %s", err)
+		return
+	}
+
+	if _, kubectlExists := exec.LookPath("kubectl"); kubectlExists != nil {
+		if resource == "" {
+			cmd := exec.Command(
+				"kubectl",
+				"get",
+				"pods",
+				"-n",
+				namespace,
+				"-o",
+				"jsonpath='{.items[*].metadata.name}'",
+			)
+			out, err := cmd.CombinedOutput()
+			if err != nil {
+				logrus.Warnf("Failed to get pod names in namespace %s: %s", namespace, err)
+			}
+			podNames = strings.Split(string(out), " ")
+		} else {
+			podNames = []string{resource}
+		}
+
+		for _, pod := range podNames {
+			cmd := exec.Command(
+				"kubectl",
+				fmt.Sprintf("--kubeconfig=%s", kubectlconfig),
+				"logs",
+				pod,
+				"-n",
+				namespace,
+			)
+			out, err := cmd.CombinedOutput()
+			if err != nil {
+				logrus.Warnf("Failed to get %s on namespace %s: %s", resource, pod, err)
+			}
+			// We still want to write the output if the resource was not found or another issue occurred as that can shed info on what's going on
+			_ = os.WriteFile(fmt.Sprintf("%s/%s-%s-logs.log", dest, resource, pod), out, os.ModePerm)
```

Comment:
why did you decide to suppress the error? If that is intentional then suggested change should work too
```suggestion
			os.WriteFile(fmt.Sprintf("%s/%s-%s-logs.log", dest, resource, pod), out, os.ModePerm)
```

Reply:
linting prevents this. Explicitly catch return values

Reply:
supressing errors is becuase this si a best-effort-catch-all util. Do not care about errors, not want to show errors to the users, just get as much logs as possible from the system.